spec/services/big_blue_button_api_spec.rb

```diff
@@ -5,6 +5,26 @@
 
 describe BigBlueButtonApi, type: :service do
   let(:bbb_service) { described_class.new }
+  let(:default_create_opts) do
+    {
+      record: true,
+      logoutURL: 'http://localhost',
+      moderatorPW: 'mp',
+      attendeePW: 'ap',
+      moderatorOnlyMessage: 'Welcome Moderator',
+      muteOnStart: false,
+      guestPolicy: 'ALWAYS_ACCEPT',
+      'meta_gl-v3-listed': 'public',
+      'meta_bbb-origin-version': 3,
+      'meta_bbb-origin': 'Greenlight'
+    }
+  end
+
+  let(:default_join_opts) do
+    {
+      join_via_html5: true
+    }
```

Comment:
Why do we need to define these? Are they not set in the service?

Reply:
For isolation purposes.
Do you need me to remove it?

Reply:
Yeah . `start_session` calls the private method, so it's fine to test them both together

Reply:
Besides that, still need the default `hashes` for the method call assertion with the current setting this  allows for duplication and make tests dependent on the underlying implementation [LINE 63](https://github.com/bigbluebutton/greenlight/pull/3355/files#diff-63e7617af1a7aeb70cbf8dcc62e3bc240d1d99ae2f3490d0b293b3710f4660d8R63).

1. We can make the methods `public` in such case so that we use them directly for assertion and improve the tests quality without having to type them statically or to keep the `let` helpers in place plus having to synchronize them with the actual returned values.
OR, 
2. We can just make the memoized helper `default_create_opts` something much simpler since it's a mock we don't need it to store all legit BBB parameters or to be a copy of the the actual `#default_create_opts` and keep the mocking.
I'll go with `2)` until I get your suggestions.
What do you think?